hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonTableOutputFormat.java

```diff
@@ -18,22 +18,311 @@
+    final CarbonLoadModel loadModel = getLoadModel(taskAttemptContext.getConfiguration());
+    loadModel.setTaskNo(new Random().nextInt(Integer.MAX_VALUE) + "");
+    final String[] tempStoreLocations = getTempStoreLocations(taskAttemptContext);
+    final CarbonOutputIteratorWrapper iteratorWrapper = new CarbonOutputIteratorWrapper();
+    final DataLoadExecutor dataLoadExecutor = new DataLoadExecutor();
+    CarbonRecordWriter recordWriter = new CarbonRecordWriter(iteratorWrapper, dataLoadExecutor);
+    new Thread() {
+      @Override public void run() {
+        try {
+          dataLoadExecutor
+              .execute(loadModel, tempStoreLocations, new CarbonIterator[] { iteratorWrapper });
+        } catch (Exception e) {
+          dataLoadExecutor.close();
+          throw new RuntimeException(e);
+        }
+      }
+    }.start();
+
+    return recordWriter;
+  }
+
+  public static CarbonLoadModel getLoadModel(Configuration conf) throws IOException {
+    CarbonLoadModel model;
+    String encodedString = conf.get(LOAD_MODEL);
+    if (encodedString != null) {
+      model = (CarbonLoadModel) ObjectSerializationUtil.convertStringToObject(encodedString);
+      return model;
+    }
+    model = new CarbonLoadModel();
+    CarbonProperties carbonProperty = CarbonProperties.getInstance();
+    model.setDatabaseName(CarbonTableOutputFormat.getDatabaseName(conf));
+    model.setTableName(CarbonTableOutputFormat.getTableName(conf));
+    model.setCarbonDataLoadSchema(new CarbonDataLoadSchema(getOrCreateCarbonTable(conf)));
+    model.setTablePath(getTablePath(conf));
+
+    setFileHeader(conf, model);
+    model.setSerializationNullFormat(conf.get(SERIALIZATION_NULL_FORMAT, "\\N"));
+    model.setBadRecordsLoggerEnable(conf.get(BAD_RECORDS_LOGGER_ENABLE, carbonProperty
+        .getProperty(CarbonLoadOptionConstants.CARBON_OPTIONS_BAD_RECORDS_LOGGER_ENABLE,
+            CarbonLoadOptionConstants.CARBON_OPTIONS_BAD_RECORDS_LOGGER_ENABLE_DEFAULT)));
+    model.setBadRecordsAction(conf.get(BAD_RECORDS_LOGGER_ACTION, carbonProperty
+        .getProperty(CarbonCommonConstants.CARBON_BAD_RECORDS_ACTION,
+            CarbonCommonConstants.CARBON_BAD_RECORDS_ACTION_DEFAULT)));
+
+    model.setIsEmptyDataBadRecord(conf.get(IS_EMPTY_DATA_BAD_RECORD, carbonProperty
+        .getProperty(CarbonLoadOptionConstants.CARBON_OPTIONS_IS_EMPTY_DATA_BAD_RECORD,
+            CarbonLoadOptionConstants.CARBON_OPTIONS_IS_EMPTY_DATA_BAD_RECORD_DEFAULT)));
+
+    model.setSkipEmptyLine(conf.get(SKIP_EMPTY_LINE,
+        carbonProperty.getProperty(CarbonLoadOptionConstants.CARBON_OPTIONS_SKIP_EMPTY_LINE)));
+
+    String complexDelim = conf.get(COMPLEX_DELIMITERS, "\\$" + "," + "\\:");
+    String[] split = complexDelim.split(",");
+    model.setComplexDelimiterLevel1(split[0]);
+    if (split.length > 1) {
+      model.setComplexDelimiterLevel1(split[1]);
+    }
+    model.setDateFormat(conf.get(DATE_FORMAT, carbonProperty
```

Comment:
can you change the code style to 
```
    model.setDateFormat(
        conf.get(
            DATE_FORMAT, 
            carbonProperty.getProperty(
                CarbonLoadOptionConstants.CARBON_OPTIONS_DATEFORMAT, 
                CarbonLoadOptionConstants.CARBON_OPTIONS_DATEFORMAT_DEFAULT)));
```
It will be more readable

Reply:
ok